test/extended/openstack/servergroup.go

```diff
@@ -79,7 +80,23 @@ var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform",
 
 			serverGroup, err := servergroups.Get(computeClient, serverGroupsWithThatName[0]).Extract()
 			o.Expect(serverGroup.Members, err).To(o.ContainElements(masterInstanceUUIDs...))
+
+			if len(serverGroup.Policies) == 1 {
+				//host_ids := make([]string, 0)
+				host_ids := make(map[string]int)
+
+				for _, server_id := range masterInstanceUUIDs {
+					server, err := servers.Get(computeClient, server_id.(string)).Extract()
+					o.Expect(err).NotTo(o.HaveOccurred())
+					host_ids[server.HostID] += 1
+				}
+				if serverGroup.Policies[0] == "strict-anti-affinity" {
+					o.Expect(host_ids).To(o.HaveLen(len(masterInstanceUUIDs)))
```

Comment:
Could you please add an error message on this check? The idea is to check that there are as many elements on host_ids as master nodes, right? I think a descriptive error message will help.

Reply:
@rlobillo Thanks , I'll add